modules/uart_io_internal.h

```diff
@@ -451,6 +479,32 @@
         #define RX_DMA_TRG          DMACTL3
         #define RX_DMA_TSEL_MASK    0x1F00
     #endif
+
+    #if(UIO_TX_DMA_CHANNEL == 0)
+        #define TX_DMA_TRG          DMACTL0
+        #define TX_DMA_TSEL_MASK    0x001F
+    #elif(UIO_TX_DMA_CHANNEL == 1)
+        #define TX_DMA_TRG          DMACTL0
+        #define TX_DMA_TSEL_MASK    0x1F00
+    #elif(UIO_TX_DMA_CHANNEL == 2)
+        #define TX_DMA_TRG          DMACTL1
+        #define TX_DMA_TSEL_MASK    0x001F
+    #elif(UIO_TX_DMA_CHANNEL == 3)
+        #define TX_DMA_TRG          DMACTL1
+        #define TX_DMA_TSEL_MASK    0x1F00
+    #elif(UIO_TX_DMA_CHANNEL == 4)
+        #define TX_DMA_TRG          DMACTL2
+        #define TX_DMA_TSEL_MASK    0x001F
+    #elif(UIO_TX_DMA_CHANNEL == 5)
+        #define TX_DMA_TRG          DMACTL2
+        #define TX_DMA_TSEL_MASK    0x1F00
+    #elif(UIO_TX_DMA_CHANNEL == 6)
+        #define TX_DMA_TRG          DMACTL3
+        #define TX_DMA_TSEL_MASK    0x001F
+    #elif(UIO_TX_DMA_CHANNEL == 6)

```

Comment:
this code is inherited from your`s. i leave it as is (copyfrom #elif(UIO_RX_DMA_CHANNEL...) , since i have no problem with this.
should i fix here something?